infrastructure/environments/update-known-hosts.sh

```diff
@@ -10,6 +10,11 @@ fi
 
 DOMAIN=$1
 IP=$(dig +short $DOMAIN)
+
```

Comment:
I added support for only using IP address as in the case of backup server, there is not a domain to refer to but we still want the IP to be in known-hosts